package/CHANGELOG

```diff
@@ -13,7 +13,7 @@ The rules for this file:
   * release numbers follow "Semantic Versioning" http://semver.org
 
 ------------------------------------------------------------------------------
-??/??/??  richardjgowers, IAlibay, tylerjereddy, xiki-tempula, lilyminium,
+04/28/21  richardjgowers, IAlibay, tylerjereddy, xiki-tempula, lilyminium,
```

Comment:
Do we really do MM/DD/YY?

Reply:
Maybe we should switch over to ISO 8601? I'd be up for it if you want to propose it in a new issue 🙃